src/memory_manager.rs

```diff
@@ -518,6 +518,30 @@ impl MemoryId {
     }
 }
 
+impl crate::Storable for MemoryId {
+    fn to_bytes(&self) -> std::borrow::Cow<[u8]> {
+        std::borrow::Cow::Owned(self.0.to_be_bytes().to_vec())
+    }
+
+    fn from_bytes(bytes: std::borrow::Cow<[u8]>) -> Self {
+        Self(u8::from_be_bytes(bytes.as_ref().try_into().unwrap()))
+    }
+}
+
+impl crate::BoundedStorable for MemoryId
+{
+    const MAX_SIZE: u32 = 1;
+    const IS_FIXED_SIZE: bool = true;
+}
+
+impl std::ops::Deref for MemoryId {
+    type Target = u8;
+
+    fn deref(&self) -> &Self::Target {
+        &self.0
+    }
+}
```

Comment:
Is there a real need to treat memory identifiers as integers?
I'd avoid allowing conversions from MemoryId and u8 unless there is a pressing need.